paddle/fluid/platform/monitor.cc

```diff
@@ -0,0 +1,30 @@
+//   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "paddle/fluid/platform/monitor.h"
+#include <utility>
+
+namespace paddle {
+namespace platform {}  // namespace platform
+}  // namespace paddle
+
+DEFINE_INT_STATUS(STAT_total_feasign_num_in_mem)
+DEFINE_INT_STATUS(STAT_gpu0_mem_size)
+DEFINE_INT_STATUS(STAT_gpu1_mem_size)
+DEFINE_INT_STATUS(STAT_gpu2_mem_size)
+DEFINE_INT_STATUS(STAT_gpu3_mem_size)
+DEFINE_INT_STATUS(STAT_gpu4_mem_size)
+DEFINE_INT_STATUS(STAT_gpu5_mem_size)
+DEFINE_INT_STATUS(STAT_gpu6_mem_size)
+DEFINE_INT_STATUS(STAT_gpu7_mem_size)
```

Comment:
Can we directly define these stats without declaration?

Reply:
The declaration may be necessary, cause it must be recognized by the file using it.